Sources/Basic/Path.swift

```diff
@@ -0,0 +1,473 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import libc
+import POSIX
+
+
+/// Path separator character (always `/`).  If a path separator occurs at the
+/// beginning of a path, the path is considered to be absolute.
+public let PathSeparatorCharacter: Character = "/"
```

Comment:
They can be private.  I didn't realize that globals should be lowercase, but I'll make them that.  I'll make the change.